dali/kernels/slice/slice_gpu.cuh

```diff
@@ -194,7 +194,10 @@ template <typename OutputType, typename InputType, int Dims>
 class SliceGPU {
  private:
   static constexpr int64_t kBlockDim = 256;
-  static constexpr int64_t kBlockSize = 64 * kBlockDim;
+  static constexpr int64_t kMinBlockSize = 4 * kBlockDim;
+  static constexpr int64_t kMaxBlockSize = 64 * kBlockDim;
+  int64 blockSize;
+
```

Comment:
```suggestion

    int64_t block_size_ = 256;
```
1. snake_case
2. trailing `_` for a member (I don't like it, but that's what we use everywhere...)
3. stick it to clock_count_ field rather than to the constants.

Reply:
Sorry for that, I had `block_count_` just below and didn't notice style difference :/